src/test/java/dk/cngroup/lentils/service/ObjectGenerator.java

```diff
@@ -20,6 +21,8 @@
 
     public static final int NUMBER_OF_CYPHERS = 5;
 
+    public static final int NUMBER_OF_HINTS_FOR_CYPHER = 3;
+
     public static final int TESTED_STAGE = 3;
```

Comment:
TESTED_STAGE a TESTED_TEAM nemaji v ObjectGeneratoru co delat. Jinak prosim o smazani volnych radku mezi vsemi konstantami. Zbytecne tim naroste velikost souboru. Mozna te mate, ze treba mezi properties, ktere maji treba annotaci @Autowired se mezera nechava, ale u techto ti tedka pisu, ze ne. Je to tak, ze standartne se mezera nedela, protoze mezera invokuje oddeleni logickych bloku, coz by se melo resit extrakci do metod. Ale pokud mame ve tride atributy s anotacemi, tak se mezera pouziva, protoze kdyz ji tam nedas, tak ocim splyvaji anotace a atributy dohromady a na prvni pohled se spatne rozeznava, jestli anotace patri k tomu radku pod nebo nad. Takze pokud budes mit nad property annotaci, tak pred anotaci bude prazdny radek a kdyz budou jen atributy nebo konstanty bez anotace, tak to muzes psat bez mezer :) Kdyztak ti to pak jeste ukazu v kodu :)